README.md

```diff
@@ -17,25 +18,267 @@ go get github.com/EdgeCast/ec-sdk-go
 ```
 
 ## Using the SDK
-Simply import the SDK and provide the API credentials provided to you. They may be an API Token or OAuth 2.0 Credentials. Examples are listed below for each feature.
+Simply import the SDK and provide the API credentials provided to you. They may 
+be an API Token or OAuth 2.0 Credentials. Examples are listed below for each 
+feature.
+
+### Customer management ###
+
+Our Customer Management service provides administrative operations to manage 
+Customer accounts and Customer User accounts for a Partner. These operations 
+allows a partner to automate administrative tasks on their customers and 
+customer user accounts. 
+
+To use this Customer Management service, use the API Token provided to you.
+
+#### Customer
+
+Customer Account Management Operations allows management of customer accounts
```

Comment:
allows you to manage customer accounts.

Reply:
updated to "allows you to manage user accounts under a (parent) customer."